membership-attribute-service/app/limit/InstanceCountOnSchedule.scala

```diff
@@ -26,7 +26,7 @@ class InstanceCountOnSchedule(stage: String)(implicit ec: ExecutionContext, syst
       .getAutoScalingGroups
       .asScala
       .toList
-      .filter(_.getAutoScalingGroupName.startsWith(s"Memb-Attributes-$stage"))
+      .filter(_.getAutoScalingGroupName.startsWith(s"Memb-Attributes-${if (stage == "UAT") "CODE" else stage}"))
```

Comment:
actually in UAT stage it is hit by both prod and code (and dev) so to be strictly correct this should get the total for PROD and CODE, and total them up.  i.e. 7.
However, we don't really care about overloading the UAT env, and it's unlikely to happen, so this fix would not cause any problem.